src/command_modules/azure-cli-resource/azure/cli/command_modules/resource/_params.py

```diff
@@ -68,6 +68,10 @@
 register_cli_argument('group deployment', 'mode', help='Incremental (only add resources to resource group) or Complete (remove extra resources from resource group)', **enum_choice_list(DeploymentMode))
 register_cli_argument('group deployment create', 'deployment_name', options_list=('--name', '-n'), required=False,
                       validator=validate_deployment_name, help='The deployment name. Default to template file base name')
+register_cli_argument('group deployment operation show', 'operation_id', options_list=('--id',), required=True, help='The id of the operation to show')
+
+register_cli_argument('group deployment operation show', 'operation_id', options_list=('--id',), help='The id of the operation to show')
```

Comment:
this is a dupe. I suggest we just expose  `operation_ids`. The convention is argument with plural name accept 1 or more than value